File: 3194-find-words-containing-character/find-words-containing-character.java
class Solution {
    public List<Integer> findWordsContaining(String[] words, char x) {
        ArrayList<Integer> a=new ArrayList<>();
        for(int i=0;i<words.length;i++)
        {   StringBuilder s=new StringBuilder();
        s.append(words[i]);
            if(s.toString().contains(String.valueOf(x)))
            {
                a.add(i);
            }
        }
        return a;
    }
}

File: 3194-find-words-containing-character/find-words-containing-character.cpp
class Solution {
public:
    vector<int> findWordsContaining(vector<string>& words, char x) {
        vector <int> ans;
        string w;
        for(int i=0;i<words.size();i++)
        { w=words[i];
            for(int j=0;j<w.size();j++)
            {
                if(w[j]==x)
                {ans.push_back(i);
                break;
                }

            }
        }
        return ans;
    }
};